src/components/EditUserProfile.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useRef } from 'react'
+import Axios from 'axios'
+
+const EditUserProfile = () => {
+  const user = JSON.parse(window.localStorage.getItem('cookie'))
```

Comment:
Using `window` during render will cause Next.js build to break. Can store a null value in state and then set the user in an effect.

Reply:
k